### PR TITLE
Documentation update for https://github.com/archivesunleashed/aut/pul…

### PR DESCRIPTION
**Pull request by ruebot**
…l/381

See https://github.com/archivesunleashed/aut/pull/381

### CODE REVIEW

**Review by ianmilligan1 — APPROVED** [review]
LGTM; once #381 is merged over on AUT we can merge this.

**Review by lintool — APPROVED** [review]

**Pull request merged**
